extensions/answer_summarizers/models_test.py

```diff
@@ -30,6 +30,15 @@ def test_requires_override_for_calculation(self):
             answer_models.BaseCalculation().calculate_from_state_answers_dict(
                 state_answers_dict={})
 
+    def test_equality_of_hashable_answers(self):
+        hashable_answer_1 = answer_models._HashableAnswer('answer_1')  # pylint: disable=protected-access
```

Comment:
Hmmm, I thought the plan was to test protected functions using public interfaces.

Reply:
I want to test the equality of hashable_answers with the __eq__ method. This is why, an exception needs to be made here

Reply:
I believe the `__eq__` method is what is actually used to implement "a == b" in Python, so you should be able to use the latter below.

Reply:
Done!